pkg/command/cdc/create.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cdc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/kn-plugin-admin/pkg"
+	typev1alpha1 "knative.dev/networking/pkg/apis/networking/v1alpha1"
+)
+
+// NewCdcCreateCommand to create cluster domain claims
+func NewCdcCreateCommand(p *pkg.AdminParams) *cobra.Command {
+	var namespace string
+	cdcCreateCommand := &cobra.Command{
+		Use:   "create",
+		Short: "create cluster domain claim",
+		Long:  "Create Knative cluster domain claim",
+		Example: `
+  # To create a cluster domain claim
```

Comment:
I'd try to mimic the docs style from `kn`. In addition maybe it'd good to explain that `--namespace` in this context means namespace that will delegated access to the particular domain.

```suggestion
  # Create a cluster domain claim for namespace 'ns1'
```

Reply:
I am following the style in this repo:

https://github.com/knative-sandbox/kn-plugin-admin/blob/main/pkg/command/registry/add.go#L50

Yes, I will add the explanation for namespace option

Reply:
Ok then, I kind of don't like, but I'll deal with it. :)